src/modules/cpu/host_fused_functions.hpp

```diff
@@ -4,12 +4,593 @@
 #include "cpu/rpp_cpu_simd.hpp"
 #include <cpu/rpp_cpu_common.hpp>
 
+/**************** color_jitter ***************/
+
+template <typename T>
+RppStatus color_jitter_host_batch(T* srcPtr, RppiSize *batch_srcSize, RppiSize *batch_srcSizeMax, T* dstPtr,
+                         Rpp32f *batch_brightness, Rpp32f *batch_contrast,
+                         Rpp32f *batch_hue, Rpp32f *batch_saturation,
+                         Rpp32u outputFormatToggle, Rpp32u nbatchSize,
+                         RppiChnFormat chnFormat, Rpp32u channel)
+{
+    if(chnFormat == RPPI_CHN_PLANAR)
+    {
+        // NOT IMPLEMENTED
+    }
+    else if(chnFormat == RPPI_CHN_PACKED)
+    {
+        omp_set_dynamic(0);
+#pragma omp parallel for num_threads(nbatchSize)
+        for(int batchCount = 0; batchCount < nbatchSize; batchCount ++)
+        {
+            Rpp64u imageDimMax = batch_srcSizeMax[batchCount].height * batch_srcSizeMax[batchCount].width;
+
+            Rpp32f hue = batch_hue[batchCount];
+            Rpp32f saturation = batch_saturation[batchCount];
+            Rpp32f brightness = batch_brightness[batchCount];
+            Rpp32f contrast = batch_contrast[batchCount];
+
+            T *srcPtrImage, *dstPtrImage;
+            Rpp32u loc = 0;
+            compute_image_location_host(batch_srcSizeMax, batchCount, &loc, channel);
+            srcPtrImage = srcPtr + loc;
+            dstPtrImage = dstPtr + loc;
+
+            color_jitter_host(srcPtrImage, batch_srcSize[batchCount], batch_srcSizeMax[batchCount], dstPtrImage, brightness, contrast, hue, saturation, chnFormat, channel);
+
+            if (outputFormatToggle == 1)
+            {
+                T *dstPtrImageUnpadded = (T*) calloc(channel * batch_srcSize[batchCount].height * batch_srcSize[batchCount].width, sizeof(T));
+                T *dstPtrImageUnpaddedCopy = (T*) calloc(channel * batch_srcSize[batchCount].height * batch_srcSize[batchCount].width, sizeof(T));
+
+                compute_unpadded_from_padded_host(dstPtrImage, batch_srcSize[batchCount], batch_srcSizeMax[batchCount], dstPtrImageUnpadded, chnFormat, channel);
```

Comment:
this seems very inefficient. Follow up to change this for all the RPP kernels

Reply:
The NCHW<->NHWC sample that is developed now for brightness will solve this

Reply:
Done. Uses the new standard for layout toggling.